src/QMCWaveFunctions/TWFFastDerivWrapper.h

```diff
@@ -57,6 +61,28 @@ class TWFFastDerivWrapper
   void addGroup(const ParticleSet& P, const IndexType groupid, SPOSet* spo);
   inline void addJastrow(WaveFunctionComponent* j) { jastrow_list_.push_back(j); };
 
+  /**
+   * @brief add a MultiSlaterDet
+   * 
+   * this is called when registering a MSD
+   * the caller should also iterate over MultiDiracDeterminants in the MSD and register the associated SPOSets
+   * 
+   * @param P. ParticleSet 
+   * @param msd. Pointer to MultiSlaterDetTableMethod
+   */
+  void addMultiSlaterDet(const ParticleSet& P, const WaveFunctionComponent* msd);
+  inline IndexType numMultiSlaterDets() const { return slaterdets_.size(); }
```

Comment:
Please start a function with a verb.

Reply:
removed with change from `vector<MSD*>` to single `MSD*`